src/main/lrsql/util.clj

```diff
@@ -19,22 +18,6 @@
 ;; Macros
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(defmacro def-hugsql-db-fns
-  "A custom version of `hugsql.core/def-db-fns` that reads .sql files
-   during compilation (instead of during runtime as in the original macro)."
-  [file]
-  (let [parsed-defs# (hug/parsed-defs-from-file file)]
-    ;; Validate defs at compile time
-    (doseq [pdef parsed-defs#]
-      (hug/validate-parsed-def! pdef))
-    ;; Runtime actions
-    `(doseq [~'exp-pdef (map hug/expand-compile-frags ~parsed-defs#)]
-       (hug/compile-exprs ~'exp-pdef)
-       (hug/dispatch-on-pdef ~'exp-pdef
-                             {} ; options - empty map is the hugsql default
-                             ~hug/intern-db-fn
-                             ~hug/intern-sqlvec-fn))))
```

Comment:
Adding for future reference, the issue https://github.com/layerware/hugsql/issues/126